### PR TITLE
Docker Compose File Restructure/Clean Up

### PR DESCRIPTION
**Pull request by michaelestrin**
Restructured compose-files directory content into releases directory. Removed Docker Compose files for California preview releases. Cleaned up documentation.

https://github.com/edgexfoundry/developer-scripts/issues/128

Signed-off-by: Michael W. Estrin <me@michaelestrin.com>

### CODE REVIEW

**Comment by michaelestrin** [comment]
Replacement PR for https://github.com/edgexfoundry/developer-scripts/pull/129

File: releases/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# edgexfoundry/developer-scripts/releases
+
+## Overview
+
+This folder contains supporting files to run various current and historical versions of EdgeX in Docker.
+
+## Table of Contents
+
+- [Nightly Build](#nightly-build)
+    - [Docker Comoose](#docker-compose-for-nightly-build)
```
Comment (lenny-intel) [inline releases/README.md]: Comoose -> Compose here and below

Comment (michaelestrin) [reply releases/README.md]: Good catch.  Corrected.

**Comment by tsconn23** [comment]
I just noticed there are links in the documentation that point to current paths in this repo. Those will need to be updated along with this PR.

Example: https://docs.edgexfoundry.org/Ch-QuickStart.html
See "latest docker-compose file here" link

@mhall119 @jpwhitemn for visibility

**Comment by tsconn23** [comment]
Ah @michaelestrin has it covered in this edgex-docs PR
https://github.com/edgexfoundry/edgex-docs/pull/12

**Pull request merged**